L1Trigger/L1THGCal/python/hgcalBackEndLayer2Producer_cfi.py

```diff
@@ -77,63 +77,73 @@
                              EGIdentification=egamma_identification_drnn_dbscan.clone())
 
 
-histoMax_C3d_params = cms.PSet(type_multicluster=cms.string('HistoMaxC3d'),
-                               dR_multicluster=cms.double(0.03),
-                               dR_multicluster_byLayer_coefficientA=cms.vdouble(),
-                               dR_multicluster_byLayer_coefficientB=cms.vdouble(),
-                               shape_threshold=cms.double(1.),
-                               minPt_multicluster=cms.double(0.5),  # minimum pt of the multicluster (GeV)
+histoMax_C3d_seeding_params = cms.PSet(type_multicluster=cms.string('Histo'),
```

Comment:
`type_multicluster` is already in the PSet `histoMax_C3d_params` so it doesn't need to be repeated here.